luminance/src/blending.rs

```diff
@@ -76,3 +76,27 @@ pub struct Blending {
   /// Destination factor.
   pub dst: Factor,
 }
+
+/// Blending configuration to represent combined or separate options.
```

Comment:
Good `enum`, I’m okay with it. However, the name could be enhanced, as `Option` has a strong meaning in Rust.

Why not… `BlendingMode`?